test/unit/header.test.js

```diff
@@ -74,6 +74,13 @@ describe('Header', function () {
     });
 
     describe('unparse', function () {
+        it('should unparse headers to a blank string for invalid inputs', function () {
+            expect(Header.unparse('')).to.be('');
+            expect(Header.unparse({ key: 'foo', value: 'bar' })).to.be('');
```

Comment:
@kunagpal why is `{ key: 'foo', value: 'bar' }` an invalid input to `Header.unparse`?